### ISSUE TITLE
Herbalism 

### ISSUE THREAD

**Issue by brad0534** [SunwellWoW/Sunwell-TBC-Bugtracker#3563]
Description: Herbs are super scarce and in some zones they don't spawn at all ive farmed for hours in every zone and 2-3 zones not 1 herb

How it works: 
its not working in most zones every herb shouldn't be super rare 

How it should work: 
the zones should have herbs = to or + than the level of the zone 

Source (you should point out proofs of your report, please give us some source): 
source is the 7 pservers ive played on and ive played since vanilla this now in BFA and know which herbs should be super rare and which ones arnt

**Comment by Wojtyas** [comment]
@Dootz if u have some free time please could you tell how many herb gobs are in specific zone in Outland? I mean in total, don't put numbers of every kind of herb.

**Comment by Reflekz** [comment]
#1238 - Brought this up a while back and never got feedback :/

I also leveled Herbing when server went live and once again , could only find herbs after restarts/crashes.
(im aware there are many people herbing but there seems to be an issue)

**Comment by Niar91** [comment]
Been leveling herbalism and this is still a problem, either you find 1-2 herbs in 1 zone or a cluster of 5-7 spawns within an arms reach to eachother. So scuffed. Please fix spawns across all zones

**Comment by AIRewan** [comment]
Yes. Felweed which is usually everywhere: I have more than 300 of that. Terocone which is mostly on terokkar forest, I found 1 /1hour which seems pretty bad.
